### PR TITLE
Add section on Data Theft to Privacy Considerations.

### PR DESCRIPTION
**Pull request by msporny**
This PR attempts to address issue #1247 by adding a section on Data Theft, and advising all roles to be careful how much data they store and how long they store it. Advice is given to regulators to play an active role in reducing the collection of PII.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/pull/1356.html" title="Last updated on Nov 21, 2023, 3:05 PM UTC (ad252fb)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/1356/b3cff22...ad252fb.html" title="Last updated on Nov 21, 2023, 3:05 PM UTC (ad252fb)">Diff</a>

### CODE REVIEW

**Review by dlongley — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -5101,6 +5101,48 @@ <h3>Sharing Information with the Wrong Party</h3>
...
+        <p>
+<a>Issuers</a> are advised to hold on to the minimum amount of data
+necessary to issue <a>verifiable credentials</a> to <a>holders</a> and
+manage the status and revocation of those credentials.
+        </p>
+        <p>
+<a>Holders</a> are advised to use implementations that appropriately
+encrypt their data in transit and at rest, and protect sensitive
+material (such as cryptographic secrets) in ways that cannot be easily
+extracted from hardware devices.
```
Comment (kdenhartog) [inline index.html:5124]:
```suggestion
extracted from hardware devices. Furthermore, it's suggested that
<a>Holders<a> hold data at the edge whenever possible in order to
reduce the likelihood of attack.
```

**Review by kdenhartog — APPROVED** [review]
LGTM - small suggestion but just a nit

**Review by brianorwhatever — APPROVED** [review]

**Review by longpd — APPROVED** [review]

**Review by decentralgabe — APPROVED** [review]

File: index.html [hunk 2]
```diff
@@ -5101,6 +5101,51 @@ <h3>Sharing Information with the Wrong Party</h3>
...
+      <section class="informative">
+        <h3>Data Theft</h3>
+
+        <p>
+The data expressed in <a>verifiable credentials</a> and
+<a>verifiable presentations</a> are valuable since they contain authentic
+statements made by trusted third parties, such as <a>issuers</a>, or
+individuals, such as <a>holders</a> and <a>subjects</a>. Storing this data
+can create sensitive data honeypots that attackers are motivated to
+break into in order to acquire and exchange it for financial gain.
```
Comment (TallTed) [inline index.html:5113]:
```suggestion
individuals, such as <a>holders</a> and <a>subjects</a>. Storing this data
can create honeypots of sensitive data that attackers are motivated to
break into in order to acquire and exchange that data for financial gain.
```

File: index.html [hunk 3]
```diff
@@ -5101,6 +5101,51 @@ <h3>Sharing Information with the Wrong Party</h3>
...
+        </p>
+        <p>
+<a>Issuers</a> are advised to hold on to the minimum amount of data
```
Comment (TallTed) [inline index.html:5116]:
```suggestion
<a>Issuers</a> are advised to retain the minimum amount of data
```

File: index.html [hunk 4]
```diff
@@ -5101,6 +5101,51 @@ <h3>Sharing Information with the Wrong Party</h3>
...
+manage the status and revocation of those credentials.
+        </p>
+        <p>
+<a>Holders</a> are advised to use implementations that appropriately
+encrypt their data in transit and at rest, and protect sensitive
+material (such as cryptographic secrets) in ways that cannot be easily
+extracted from hardware devices. Furthermore, it is suggested that
+<a>holders<a> store and manipulate their data on devices that they control,
+away from centralized systems, in order to reduce the likelihood of attack
+on their data, or large-scale theft if an attack is successful.
```
Comment (TallTed) [inline index.html:5127]:
```suggestion
<a>Holders</a> are advised to use implementations that appropriately
encrypt their data both in transit and at rest, and protect sensitive
material (such as cryptographic secrets) in ways that cannot be easily
extracted from hardware devices. Furthermore, it is suggested that
<a>holders<a> store and manipulate their data only on devices that they
control, away from centralized systems, to reduce the likelihood of
attack on their data, or large-scale theft if an attack is successful.
```

File: index.html [hunk 5]
```diff
@@ -5101,6 +5101,51 @@ <h3>Sharing Information with the Wrong Party</h3>
...
+        </p>
+        <p>
+<a>Verifiers</a> are advised to only ask for data necessary for a particular
+transaction and to not hold on to any data beyond the needs of any particular
```
Comment (TallTed) [inline index.html:5131]:
```suggestion
transaction and to not retain any data beyond the needs of any particular
```

File: index.html [hunk 6]
```diff
@@ -5101,6 +5101,51 @@ <h3>Sharing Information with the Wrong Party</h3>
...
+privacy-preserving mechanisms can be used to achieve similar levels of
+enforcement and audit capabilities. For example, regulations that insist on the
+collection and long-term storage of personally identifiable information for the
+purposes of auditing can create harms to individuals and organizations if that
+same information is compromised and accessed by an attacker. The technologies
+described by this specification enable <a>holders</a> to more readily prove
+attributes about themselves and others, reducing the need for long-term data
+archival by <a>verifiers</a>. Alternatives include keeping logs that the
+information was collected and checked along with random audits to ensure that
+compliance regimes are operating as expected.
```
Comment (TallTed) [inline index.html:5145]:
```suggestion
Regulators are advised to rethink audit requirements such that more
privacy-preserving mechanisms can be used to achieve similar levels of
enforcement and audit capabilities. For example, audit-focused regulations
that insist on collection and long-term retention of personally identifiable
information can cause harm to individuals and organizations if that same
information is compromised and accessed by an attacker. The technologies
described by this specification enable <a>holders</a> to more-readily prove
attributes about themselves and others, reducing the need for long-term data
retention by <a>verifiers</a>. Alternatives include keeping logs that the
information was collected and checked, as well as random tests to ensure
that compliance regimes are operating as expected.
```

**Review by TallTed — CHANGES_REQUESTED** [review]
Editorial language tweaks

**Comment by msporny** [comment]
Editorial, multiple reviews, changes requested and made, no objections, merging.

**Pull request merged**